pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -703,7 +703,7 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podContainerC
 	enforceCPULimits := m.cpuCFSQuota
 	if utilfeature.DefaultFeatureGate.Enabled(features.DisableCPUQuotaWithExclusiveCPUs) && m.containerManager.PodHasExclusiveCPUs(pod) {
 		enforceCPULimits = false
-		klog.V(2).InfoS("Disabled CFS quota", "pod", klog.KObj(pod))
+		klog.V(5).InfoS("Disabled CFS quota", "pod", klog.KObj(pod))
```

Comment:
likewise, per discussion in https://github.com/kubernetes/kubernetes/pull/127525/files#r1940278408 this log is expected to happen rarely and is expected to tell us the fix took place. So I think it should be kept V=2 (my previous comment was too broad).